tests/postman-collection/published-cohort-definitions.postman_collection.json

```diff
@@ -0,0 +1,163 @@
+					"raw": "{\"query\":\"query PublishedCohortLibraryGetAll {\\n    PublishedCohortLibraryGetAll {\\n        authors\\n        clinicalAtRiskGroupsText\\n        id\\n        commissioner\\n        demographicsText\\n        description\\n        disclaimerText\\n        fixedDateReference\\n        name\\n        purpose\\n        shortName\\n        summary\\n        urlSlug\\n    }\\n}\"\n}"
+				},
+				"url": {
+					"raw": "https://internal-dev.api.service.nhs.uk/caas-published-cohort-definitions/api",
+					"protocol": "https",
+					"host": [
+						"internal-dev",
+						"api",
+						"service",
+						"nhs",
+						"uk"
+					],
+					"path": [
+						"caas-published-cohort-definitions",
+						"api"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "PublishedCohortLibraryGetAll - SANDBOX",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "graphql",
+					"graphql": {
+						"query": "query PublishedCohortLibraryGetAll {\n    PublishedCohortLibraryGetAll {\n        authors\n        clinicalAtRiskGroupsText\n        id\n        commissioner\n        demographicsText\n        description\n        disclaimerText\n        fixedDateReference\n        lastUpdated\n        name\n        purpose\n        shortName\n        summary\n        urlSlug\n    }\n}",
+						"variables": ""
+					}
+				},
+				"url": {
+					"raw": "https://internal-dev-sandbox.api.service.nhs.uk/caas-published-cohort-definitions/api",
+					"protocol": "https",
+					"host": [
+						"internal-dev-sandbox",
+						"api",
+						"service",
+						"nhs",
+						"uk"
+					],
+					"path": [
+						"caas-published-cohort-definitions",
+						"api"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "PublishedCohortLibraryGetAll",
+			"request": {
+				"auth": {
+					"type": "apikey",
+					"apikey": [
+						{
+							"key": "value",
+							"value": "lVlQRHlM4M111q8fnmLBe201HAWMNQJH16SU8Q4C",
+							"type": "string"
+						},
+						{
+							"key": "key",
+							"value": "x-api-key",
+							"type": "string"
+						}
+					]
+				},
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "graphql",
+					"graphql": {
+						"query": "query PublishedCohortLibraryGetAll {\n    PublishedCohortLibraryGetAll {\n        authors\n        clinicalAtRiskGroupsText\n        id\n        commissioner\n        demographicsText\n        description\n        disclaimerText\n        fixedDateReference\n        name\n        purpose\n        shortName\n        summary\n        urlSlug\n    }\n}",
+						"variables": ""
+					}
+				},
+				"url": {
+					"raw": "https://internal-dev.api.service.nhs.uk/caas-published-cohort-definitions/api",
+					"protocol": "https",
+					"host": [
+						"internal-dev",
+						"api",
+						"service",
+						"nhs",
+						"uk"
+					],
+					"path": [
+						"caas-published-cohort-definitions",
+						"api"
+					]
+				}
+			},
+			"response": []
+		}
+	]
+}
```

Comment:
Are we going to have the introspection enabled? If yes, then we don't need to add a separate schema. If the introspection is going to be disabled we need the schema

Reply:
probably we need some instructions on how to run locally, in Postman, the tests with the Schema.